src/main/webapp/i18n/en/dragAndDropQuestion.json

```diff
@@ -30,6 +30,8 @@
             "addMappingsInstructions": "Drag items onto drop locations to define correct solutions.",
             "addDragItemPicture": "Add Picture Drag Item",
             "addDragItemText": "Add Text Drag Item",
+            "cutImagesFromDropLocation": "Cut Images from Drop Locations",
```

Comment:
```suggestion
            "cutImagesFromDropLocation": "Cut images from drop locations",
```

Reply:
This is a button text and the capitalization would look off with `addDragItemPicture ` and `addDragItemText `